sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/HttpRequest.java

```diff
@@ -157,21 +160,21 @@ public BinaryData getBody() {
     }
 
     /**
-     * Set request content.
-     * <p>
-     * If provided content has known length, i.e. {@link BinaryData#getLength()} returns non-null then Content-Length
-     * header is updated. Otherwise, if provided content has unknown length, i.e. {@link BinaryData#getLength()} returns
-     * null then the caller must set the Content-Length header to indicate the length of the content, or use
-     * Transfer-Encoding: chunked.
+     * Set the {@link HttpRequest request} content.
      *
-     * @param content The request content.
+     * <p>If the provided content has a known length (i.e. {@link BinaryData#getLength()} returns non-null), then the
+     * {@code Content-Length} header is updated. Otherwise, if the provided content has an unknown length
+     * (i.e. {@link BinaryData#getLength()} returns {@code null}), then the caller must set the {@code Content-Length}
+     * header to indicate the length of the content, or use the {@code transfer-encoding: chunked} header.</p>
      *
-     * @return This HttpRequest.
+     * @param content The {@link HttpRequest request} content.
+     *
+     * @return The updated {@link HttpRequest}.
      */
     public HttpRequest setBody(BinaryData content) {
         this.body = content;
 
-        // TODO (alzimmer): should the Content-Length header be removed if content is null?
+        // TODO (alzimmer): Should the Content-Length header be removed if the content is null?
```

Comment:
Avoid making unrelated changes in your PR - it makes git blame and history more convoluted.